tuplex/core/src/physical/BlockBasedTaskBuilder.cc

```diff
@@ -58,12 +58,23 @@ namespace tuplex {
             FunctionType* read_block_type = FunctionType::get(env().i64Type(), {env().i8ptrType(),
```

Comment:
let's maybe create a helper function next to where the func tyoe read_block_f is declared (CodegenHelper?) to create the type of the read_block_function.

Reply:
same for the argnames

Reply:
and some test that length of argnames matches the functype.